diff -urN matplotlib-0.99.1.1/src/_png.cpp matplotlib-0.99.1.1.new/src/_png.cpp
--- matplotlib-0.99.1.1/src/_png.cpp	2009-08-01 21:14:01.000000000 +0200
+++ matplotlib-0.99.1.1.new/src/_png.cpp	2010-02-11 20:40:32.379221907 +0100
@@ -290,7 +290,7 @@
 
   //free the png memory
   png_read_end(png_ptr, info_ptr);
-  png_destroy_read_struct(&png_ptr, &info_ptr, png_infopp_NULL);
+  png_destroy_read_struct(&png_ptr, &info_ptr, NULL);
   fclose(fp);
   for (row = 0; row < height; row++)
     delete [] row_pointers[row];
